webserver/tyrus/src/main/java/io/helidon/webserver/tyrus/TyrusReaderSubscriber.java

```diff
@@ -63,16 +63,15 @@ public void onSubscribe(Flow.Subscription subscription) {
 
     @Override
     public void onNext(DataChunk item) {
+        // Copy underlying buffer into a ByteBuffer and release DataChunk
+        ByteBuffer byteBuffer = ByteBuffer.wrap(item.bytes());
```

Comment:
Isn't there a way to know that the WebSocket handler is done and that the corresponding DataChunk can be released ?
Copying the data may have a performance cost right ?

Reply:
Yes, there is cost. Actually, I think there's a better way to do this ... Will try.

Reply:
Latest version avoids copying and is more draconian. I think there was a problem with subscriptions that was likely introduced when we switched to multiple arrays in a `DataChunk` --which, of course, would not be easily testable under normal conditions.